pages/common/yesod.md

```diff
@@ -1,9 +1,9 @@
 # yesod
 
-> A framework to develop Haskell web applications.
-> All yesod commands are invoked through the `stack` project manager.
+> Helper tool for the Yesod Haskell-based web framework.
```

Comment:
Yeah, I considered simply removing the reference to Haskell to make the sentence structure feel more natural. We could do that, but your proposal sounds actually pretty OK to me, if we use a comma instead of a semicolon.